wgs/wg-serving/wg-charter-serving.md

```diff
@@ -0,0 +1,58 @@
+# WG Serving Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
```

Comment:
Trying to understand the subprojects of this WG. Currently, I think it's kfserving, what's the status of seldon? If we have more serving projects in the future. For example, bentoML wants to integrate more with KFServing or other components. they are all under this WG, right?

Reply:
Seldon is virtually part of every dicussion we have, courtesy @cliveseldon being there. We are striving to do tighter collaboration with Seldon as part of WG, and though not listed explicitly here, we will be adding more on it soon.

I will list both Seldon and BentoML currently under the cross-cutting concerns here as well.